Sources/Build/BuildPlan.swift

```diff
@@ -717,8 +724,13 @@ public final class SwiftTargetBuildDescription {
             // can construct the linker flags. In the future we will use a generated
             // code stub for the cases in which the linker doesn't support it, so that
             // we can rename the symbol unconditionally.
-            if buildParameters.linkerFlagsForRenamingMainFunction(of: target) != nil {
-                args += ["-Xfrontend", "-entry-point-function-name", "-Xfrontend", "\(target.c99name)_main"]
+            // No `-` for these flags because the set of Strings in driver.supportedFrontendFlags do
+            // not have a leading `-`
+            let flags: Set = ["Xfrontend", "entry-point-function-name"]
+            if try SwiftTargetBuildDescription.checkSupportedFrontendFlags(flags: flags, fs: self.fs) {
```

Comment:
It doesn't seem 100% right to error out if we can't determine whether the frontend supports renaming; instead it seems to me that we should treat that case as if it didn't.  One option would be to change the function to return false instead of throwing, another would be to deal with the error here and treat it as not supporting it.